### PR TITLE
Import css screens

### PR DESCRIPTION
**Pull request by kart-c**
## Imported CSS screens 

Note - Images are not added yet. It will be added later

### CODE REVIEW

File: src/Components/Card/Card.jsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import './Card.css';
+
+const Card = () => {
+	return (
+		<article class="card product-card card-shadow">
```
Comment (RahullRawat) [inline src/Components/Card/Card.jsx]: you can change class to className

File: src/Components/Header/Header.jsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import './Header.css';
+
+const Header = () => {
+	return (
+		<header class="header">
```
Comment (RahullRawat) [inline src/Components/Header/Header.jsx]: Here also you can change class to className

File: src/Pages/Home/Components/Carousel/Carousel.jsx [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import './Carousel.css';
+
+const Carousel = () => {
+	return (
+		<div class="lp-carousel-container">
```
Comment (RahullRawat) [inline src/Pages/Home/Components/Carousel/Carousel.jsx]: here

File: src/Pages/Home/Components/Categories/Categories.jsx [hunk 4]
```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import './Categories.css';
+
+const Categories = () => {
+	return (
+		<div class="category-list">
```
Comment (RahullRawat) [inline src/Pages/Home/Components/Categories/Categories.jsx]: here

File: src/Pages/Home/Components/Footer/Footer.jsx [hunk 5]
```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import './Footer.css';
+
+const Footer = () => {
+	return (
+		<footer class="lp-footer">
```
Comment (RahullRawat) [inline src/Pages/Home/Components/Footer/Footer.jsx]: here

File: src/Pages/Login/Login.jsx [hunk 6]
```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { Header } from '../../Components';
+import './Login.css';
+
+const Login = () => {
+	return (
+		<>
+			<Header />
+			<main>
+				<div class="login-container">
```
Comment (RahullRawat) [inline src/Pages/Login/Login.jsx]: here

File: src/Pages/Login/Login.jsx [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+					<h3>Login</h3>
+					<div class="input-container">
+						<label for="email">Email Address *</label>
```
Comment (RahullRawat) [inline src/Pages/Login/Login.jsx]: you can change for to htmlFor

File: src/Pages/Products/Components/Filter/Filters.jsx [hunk 8]
```diff
@@ -0,0 +1,68 @@
...
+
+const Filters = () => {
+	return (
+		<aside class="pl-aside">
+			<div class="aside-heading">
+				<h4>Filters</h4>
+				<button>Clear</button>
+			</div>
+			<div class="aside-price">
+				<label for="price">Price</label>
```
Comment (RahullRawat) [inline src/Pages/Products/Components/Filter/Filters.jsx]: here

File: src/Pages/Products/Products.jsx [hunk 9]
```diff
@@ -0,0 +1,33 @@
...
+							Products
+						</a>
+					</div>
+					<Card />
+					<Card />
+					<Card />
+					<Card />
+				</section>
+				<footer class="filter-footer">
+					<button class="btn-primary">Filters</button>
```
Comment (RahullRawat) [inline src/Pages/Products/Products.jsx]: here

File: src/Pages/Signup/Signup.jsx [hunk 10]
```diff
@@ -0,0 +1,44 @@
...
+const Signup = () => {
+	return (
+		<>
+			<Header />
+			<main>
+				<div class="login-container">
+					<h3>Signup</h3>
+					<div class="input-container">
+						<label for="email">Full Name *</label>
+						<input type="email" id="email" name="email" placeholder="Enter your full name" />
```
Comment (RahullRawat) [inline src/Pages/Signup/Signup.jsx:13]: here

**Review by RahullRawat — CHANGES_REQUESTED** [review]
Other than suggested all good

File: src/Components/Card/Card.jsx (hunk 1 above)
Comment (mohammed-sas) [inline src/Components/Card/Card.jsx]: Replace class with className

**Review by mohammed-sas — CHANGES_REQUESTED** [review]
Kindly do the suggested changes

File: src/Components/Card/Card.jsx (hunk 1 above)
Comment (kart-c) [reply src/Components/Card/Card.jsx]: done thank you for noticing

Comment (kart-c) [reply src/Components/Card/Card.jsx]: Thank you Mohammed, I have replaced it

**Review by RahullRawat — APPROVED** [review]

**Review by mohammed-sas — APPROVED** [review]

**Pull request merged**